src/Menu/__snapshots__/Menu.test.jsx.snap

```diff
@@ -95,7 +95,6 @@ exports[`Menu Item renders correctly renders as expected with menu items 1`] = `
   <a
     className="pgn__hyperlink default-link standalone-link pgn__menu-item"
     href="https://en.wikipedia.org/wiki/Hyperlink"
-    onClick={[Function]}
```

Comment:
While I think it's absolutely great that we don't have "bonus" `onClick`s anymore, it's not clear to me what caused this change. Any insight into this would be awesome!

Reply:
The removal of the "bonus" `onClick` was caused by changing the `defaultProps.onClick` to `undefined` instead of the no-op function `() => {}`. I'm not sure having the no-op function is providing much value, so opted to remove it since its only a pass-thru prop, and isn't a breaking change.

For example, we don't do any conditional checks before calling it within `Hyperlink` (e.g., nothing like, `onClick.?(e)` implemented).